docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,81 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+
+Use the segmenter to identify well separated objects above a flat plane.
+Configure the `with_geometries` attribute according to whether you've [configured the frame system](/services/frame-system/#configuration) to provide intrinsic parameters:
```

Comment:
```suggestion
First, choose whether you want t configure the `with_geometries` attribute to provide intrinsic parameters from [configuring the frame system](/services/frame-system/#configuration).

The options are as follows:
```
I think I see why you're confused-- removed so it's not directing to configure-- what i mean to say is that you have to choose whether you want to or not as the beginning of the configuration process, bc this attribute should be determined  based on what you have done in the past -- configured the frame system or not. Does that make any sense? 

I would just remove this full section, but I think it's important to have the options listed so people can see immediately that their choice to configure frame system or not/selection for this attribute will then have this effect on what your segmenter returns